content/02.intro.md

```diff
@@ -36,30 +37,38 @@ Easy-to-use software packages have brought the techniques of the field out of th
-In this review, we do not exhaustively discuss the different types of deep neural network architectures; an overview of the principal terms used herein is given in Table @tbl:glossary.
-A recent book from Goodfellow et al. covers these in detail [@url:http://www.deeplearningbook.org/].
 Finally, the larger datasets now available are also sufficient for fitting the many parameters that exist for deep neural networks.
 The convergence of these factors currently makes deep learning extremely adaptable and capable of addressing the nuanced differences of each domain to which it is applied.
 
+![Neural networks come in many different forms. Left: a key for the various types of nodes used in neural networks. Simple FFNN: a feed forward neural networks in which inputs are connected via some function to an output node and the model is trained to produce some output for a set of inputs. MLP: the multi-layer perceptron is a feed forward neural network in which there is at least one hidden later between the input and output nodes. CNN: the convolutional neural network is a feed forward neural network in which the inputs are grouped spatially into hidden nodes. In the case of this example, each input node is only connected to hidden nodes alongside their neighboring input node. Deep autoencoder: a type of MLP in which the neural network is trained to produce an output that matches the input to the network. Deep RNN: a deep recurrent neural network is used to allow the neural network to retain memory over time or sequential inputs. ](https://user-images.githubusercontent.com/542643/34907749-3115860a-f851-11e7-99a7-03d4d8a27802.png){#fig:nn-petting-zoo}
+
+This review discusses recent work in the biomedical domain and most successful applications select neural network architectures that are well suited to the problem at hand.
+We sketch out a few simple example architectures in Figure @fig:nn-petting-zoo.
+If data have a natural adjacency structure, convolutional neural network (CNN) can take advantage of that structure by emphasizing local relationships, especially when convolutional layers are used in early layers of the neural network.
+Other neural network architectures such as autoencoders require no labels and are now regularly used for unsupervised tasks.
+In this review, we do not exhaustively discuss the different types of deep neural network architectures; an overview of the principal terms used herein is given in Table @tbl:glossary.
+A recent book from Goodfellow et al. covers neural network architectures in detail [@url:http://www.deeplearningbook.org/].
+
 | Term | Definition |
 |----------------------------|-------------------------------------------------------------------------------------------|
-| Neural network  (NN) | Machine-learning approach where inputs are fed into one or more hidden layers, producing an outer layer |
-| Deep learning (DL) approach | NN with multiple hidden layers |
```

Comment:
I agree `DL` can be removed.  Please expand `DL` to `deep learning` in the RBM row below and the Categorize sentence `Wang et al. [@arxiv:1705.02315] proposed to build predictive DL models`.